pkg/sql/compile/compile.go

```diff
@@ -4016,6 +4016,9 @@ func (c *Compile) generateNodes(n *plan.Node) (engine.Nodes, []any, []types.T, e
 		if err != nil {
 			return nil, nil, nil, err
 		}
+		if n.TableDef.Name == "defect_history" {
```

Comment:
You cannot do this.    This means ALL queries touching this table will generate a log line.   Which  maybe way too excessive.

Reply:
And if there are other customer who has a table named defect_history, they will log too.

The right way is always have a session level configuration parameter, if we want to generate this log, we turn it on in a session, run query,  and look at log, then turn it off.

Reply:
Yeah ,  i' ll remove all the logs after verify this pr on guangheng product env.